stock_move_location/README.rst

```diff
@@ -0,0 +1,21 @@
+**This file is going to be generated by oca-gen-addon-readme.**
```

Comment:
I'am not totally sure but I things this file is useless

Reply:
it's fine, and in the ctx of a pr like this it tells reviewers "don't check here, go to /readme" and tells bots/travis/etc to not complain about missing files